### DISCUSSION TITLE
使用frp将本地mc服务器映射到公网 - WxW

### DISCUSSION THREAD

**Discussion by giscus** [Whbbit1999/wxw-blog-comments#7] [General]
# 使用frp将本地mc服务器映射到公网 - WxW

最近买了一个迷你主机，想着用来搭建一个 minecraft 服务器用来和朋友们一起联机，搭建好了才发现好像没什么玩 minecraft 的朋友了。

http://localhost:4000/2023/03/31/%E4%BD%BF%E7%94%A8frp%E5%B0%86%E6%9C%AC%E5%9C%B0mc%E6%9C%8D%E5%8A%A1%E5%99%A8%E6%98%A0%E5%B0%84%E5%88%B0%E5%85%AC%E7%BD%91/

**Comment by Whbbit1999** [comment]
测试了一下效果，延迟很高。有点尴尬

**Comment by weekdaycare** [comment]
之前用树莓派弄了内网穿透放在家里挂着，结果忘记设置静态ip了，家里网断一次就噶了

**Reply by Whbbit1999** [reply]
😄我还没设置呢，赶紧设置一下先
